src/Docker/JobScopedEncryptor.php

```diff
@@ -35,7 +36,9 @@ public function encrypt($data)
         transferable. For both types of projects, the configId is ignored, because currently this method is
         used only to encrypt state. The question whether configId should be included in the state cipher or not
         is to be resolved https://keboola.atlassian.net/browse/PST-960 . */
-        if (in_array(self::PROTECTED_DEFAULT_BRANCH_FEATURE, $this->projectFeatures, true)) {
+        if (in_array(self::PROTECTED_DEFAULT_BRANCH_FEATURE, $this->projectFeatures, true)
+            || in_array(self::STORAGE_BRANCHES_FEATURE, $this->projectFeatures, true)
+        ) {
```

Comment:
Tady to podle mě update nepotřebuje a možu zrušit šifry sů v brančích přenositelné a tady to na branch storage nešahá. Mohl by jen posvětit @odinuv třeba.

Reply:
pokropil jsem to včera na callu svěcenou vodou, že tu bejt nemusí